pkg/controller/migcluster/remote_watch.go

```diff
@@ -64,6 +66,16 @@ func StartRemoteWatch(r *ReconcileMigCluster, config RemoteManagerConfig) error
 		log.Error(err, "[rWatch] Unable add Velero APIs to scheme")
 		return err
 	}
+	log.Info("[rWatch] Adding OpenShift imagestream to scheme")
+	if err := imagestreamregister.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
I suspect adding all OpenShift APIs would fail in the same place that adding just the image APIs would fail -- because that class is re-adding SecretList, which is causing errors for us.